python/ray/tests/test_basic.py

```diff
@@ -2801,3 +2801,32 @@ def test_redis_lru_with_set(ray_start_object_store_memory):
 
     # Now evict the object from the object store.
     ray.put(x)  # This should not crash.
+
+
+@pytest.mark.parametrize("two_node_cluster", [{"num_cpus": 10}], indirect=True)
+def test_call_shutdown_in_remote_function(two_node_cluster):
+    num_cpus = 10
+
+    @ray.remote
+    class Actor(object):
+        def shutdown(self):
+            time.sleep(1)
+            ray.shutdown()
+
+    actors = [Actor.remote() for i in range(num_cpus)]
+    [actors[i].shutdown.remote() for i in range(num_cpus)]
+
+    def function(num):
+        print(num)
```

Comment:
don't use print? although it doesn't really print anything, it'd still be better to avoid using print in code.

Reply:
Thanks, I used an empty function instead.